LICENSE.md

```diff
@@ -0,0 +1,702 @@
+# Docs-dangit - Code search engine
+
+Copyright 2023 by the contributors
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
```

Comment:
If we're adding v3 of the GPL, shouldn't we then also say v3 here?

Reply:
Yes, fixed.